### ISSUE TITLE
Logout doesn't work

### ISSUE THREAD

**Issue by FlavioFabbrizi** [andissanayake/UnifiedApp#15]
Hi, if I try to test your sample with postman. logout doesn't work.
1) invoke Register user
2) invoke Login --> return token: use the token in the methods below
3) invoke Profile --> it return my email
4) invoke logout --> return true
5) invoke Profile --> it return my email  !!!! it should be unauthorized

**Comment by andissanayake** [comment]
You are correct, this logout has zero impact on the token, The token will work properly until it expires. Please go through this StackOverflow thread as well [Invalidating JSON Web Tokens](https://stackoverflow.com/questions/21978658/invalidating-json-web-tokens). I'll plan this as a new improvement. We will maintain blacklist tokens in a table.
